mlflow/models/evaluation/base.py

```diff
@@ -653,40 +642,64 @@ def _evaluate(
-                       capable of evaluating the specified model on the specified
-                       dataset are used. The default evaluator can be referred to
-                       by the name 'default'. If this argument is unspecified, then
-                       fetch all evaluators from the registry.
-    :param evaluator_config: A dictionary of additional configurations to supply
-                             to the evaluator. If multiple evaluators are
-                             specified, each configuration should be supplied as
-                             a nested dictionary whose key is the evaluator name.
-    :return: An :py:class:`mlflow.models.evaluation.EvaluationDataset` instance containing
+
+    :param dataset_name: (Optional) The name of the dataset, must not contain double quotes (“).
+                         the name is logged to the `mlflow.datasets` tag. If not specified, the
+                         dataset hash is used as the dataset name.
+
+    :param dataset_path: (Optional) The path to a serialized DataFrame (e.g. a delta table,
+                         parquet file), must not contain double quotes (“). If specified,
+                         the path is logged to the `mlflow.datasets` tag.
+
+    :param feature_names: (Optional) If the `data` argument is a feature data numpy array or list,
+                          `feature_names` is a list of the feature names for each feature, if
+                          `None`, then the `feature_names` are generated using the format
+                          `feature_{feature_index}`. If the `data` argument is a Pandas DataFrame
+                          or a Spark DataFrame, `feature_names` is a list of the names of the
+                          feature columns in the DataFrame. If `None`, then all columns except
+                          the label column are regarded as feature columns.
+
+    :param evaluators: The name of the evaluator to use for model evaluations, or a list of
+                       evaluator names. If unspecified, all evaluators capable of evaluating the
+                       specified model on the specified dataset are used. The default evaluator
+                       can be referred to by the name 'default'. If this argument is unspecified,
+                       then fetch all evaluators from the registry. To get all available
+                       evaluators, call :py:func:`mlflow.models.list_evaluators`
+
+    :param evaluator_config: A dictionary of additional configurations to supply to the evaluator.
+                             If multiple evaluators are specified, each configuration should be
+                             supplied as a nested dictionary whose key is the evaluator name.
+
+    :return: An :py:class:`mlflow.models.EvaluationResult` instance containing
              evaluation results.
 
     The default evaluator supports the 'regressor' and 'classifer' model types.
```

Comment:
Can we add a header to this section about default evaluator? E.g.

```
Default Evaluator behavior
--------------------------
...
```

Can we also turn the items into bullet points by adding `-` for readability? E.g

```
      - The default evaluator supports the 'regressor' and 'classifer' model types. 
      - For both the 'regressor' and 'classifer' types, the default evaluator will generate model
         summary plots and feature importance plots generated by shap explainer.
```